lemur/plugins/lemur_acme/challenge_types.py

```diff
@@ -192,17 +195,19 @@ def create_certificate(self, csr, issuer_options):
         authority = issuer_options.get("authority")
         create_immediately = issuer_options.get("create_immediately", False)
         acme_client, registration = self.acme.setup_acme_client(authority)
+        domains = self.acme.get_domains(issuer_options)
         dns_provider = issuer_options.get("dns_provider", {})
 
         if dns_provider:
-            dns_provider_options = dns_provider.options
+            for domain in domains:
+                # Currently, we only support specifying one DNS provider per certificate, even if that
```

Comment:
This is lifted from [`get_ordered_certificate()`](https://github.com/Netflix/lemur/blob/master/lemur/plugins/lemur_acme/plugin.py#L85), the roughly analogous (albeit it's for pending certs) method that celery calls.